Dockerfile

```
@@ -27,6 +27,8 @@ RUN make cli
 # release image with kubectl + kuttl
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 
+LABEL org.opencontainers.image.source="https://github.com/kudobuilder/kuttl/"
```

```suggestion
LABEL org.opencontainers.image.source="https://github.com/kudobuilder/kuttl"
```
I am not sure what's correct, but it seems to me that it's more common to not include a trailing slash when using this label.

---

thanks @erikgb !